web/src/test/java/se/inera/intyg/webcert/web/web/controller/integrationtest/facade/PdlIT.java

```diff
@@ -456,12 +456,31 @@ public void shallPdlLogReadActivityWithSjfWhenFetchingCertificateOnDifferentCare
             assertPdlLogMessage(ActivityType.READ, testSetup.certificateId(), ACTIVITY_ARGS_READ_SJF);
         }
 
-        @Disabled("Until it has been implemented")
         @Test
         public void shallPdlLogSendActivityWhenSendingCertificate() {
-//        Log.activity.activityType = "Utskrift"
-//        Log.activity.activityArg = "Intyg skickat till mottagare [mottagare]"
-            fail();
+            final var testSetup = TestSetup.create()
+                .certificate(
+                    LisjpEntryPoint.MODULE_ID,
+                    "1.2",
+                    ALFA_VARDCENTRAL,
+                    DR_AJLA,
+                    ATHENA_ANDERSSON.getPersonId().getId()
+                )
+                .clearPdlLogMessages()
+                .login(DR_AJLA_ALFA_VARDCENTRAL)
+                .setup();
+
+            certificateIdsToCleanAfterTest.add(testSetup.certificateId());
+
+            given()
+                .pathParam("certificateId", testSetup.certificateId())
+                .when()
+                .get("/moduleapi/intyg/{certificateId}/send")
```

Comment:
Should use the CertificateController´s send-endpoint. Same as the send-test in LisjpIT.

            final var response = given()
                .pathParam("certificateId", testSetup.certificateId())
                .contentType(ContentType.JSON)
                .when().post("api/certificate/{certificateId}/send")
                .then().extract().response();